.gitignore

```diff
@@ -1,3 +1,4 @@
 vendor
 bin
 composer.lock
+.idea
```

Comment:
Can you put it in your own `.git/info/exclude` file?

Reply:
Can... generally don't b/c some projects do check this in. 👍